src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useCallback, useContext, useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { TodoEditForm } from './TodoEditForm/TodoEditForm';
+import { TodoListContext } from '../../context/TodoListContext';
+
+type Props = {
+  todo: Todo,
+  isProcessing: boolean,
+};
+
+export const TodoItem: React.FC<Props> = React.memo(({
+  todo,
+  isProcessing = false,
+}) => {
+  const [isEditing, setIsEditing] = useState(false);
+  const [todoTitle, setTodoTitle] = useState('');
```

Comment:
Initialize the todoTitle state with the title of the todo instead of an empty string.